client/src/app/users/user-card.component.spec.ts

```diff
@@ -1,40 +1,6 @@
-import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
-
-import { UserCardComponent } from './user-card.component';
-import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
-import { MatCardModule } from '@angular/material/card';
-
-describe('UserCardComponent', () => {
-  let component: UserCardComponent;
-  let fixture: ComponentFixture<UserCardComponent>;
-
-  beforeEach(waitForAsync(() => {
-    TestBed.configureTestingModule({
-      imports: [
-        BrowserAnimationsModule,
-        MatCardModule
-      ],
-      declarations: [ UserCardComponent ]
-    })
-    .compileComponents();
-  }));
-
-  beforeEach(() => {
-    fixture = TestBed.createComponent(UserCardComponent);
-    component = fixture.componentInstance;
-    component.user = {
-      _id: 'chris_id',
-      name: 'Chris',
-      age: 25,
-      company: 'UMM',
-      email: 'chris@this.that',
-      role: 'admin',
-      avatar: 'https://gravatar.com/avatar/8c9616d6cc5de638ea6920fb5d65fc6c?d=identicon'
-    };
-    fixture.detectChanges();
-  });
-
-  it('should create', () => {
-    expect(component).toBeTruthy();
```

Comment:
I think testing that the component does actually initialize is still important. This test still catches any issues that could be there in terms of creating the component.

Reply:
The test coverage suggests that other things are at least indirectly running all this code. Do you think that handles this, or do we need the separate test?

The problem I had with this was there was a _lot_ of code (you have to do all that ceremony to set up the tests), and then there was pretty much no payoff. I tried (spent too long, actually) figuring out a way to make a _meaningful_ test here, but just couldn't figure something out.

Thoughts?

Reply:
I suppose the component is being created in the user list where its used but to me that's not really testing _this_ component.

Reply:
I get that. I think that for the moment I'm going to let this go, but I'll create an issue to encourage folks to return to this question later.